dev-requirements.txt

```diff
@@ -1,5 +1,6 @@
 alabaster==0.7.9
 coverage==4.3.1
+coverage_pth==0.0.1
```

Comment:
Yep. Which only does anything if the COVERAGE_PROCESS_STARTUP environment variable is set.